crates/sui-graphql-rpc/src/data/pg.rs

```diff
@@ -16,6 +16,7 @@ use sui_indexer::indexer_reader::IndexerReader;
 
 use tracing::error;
 
+#[derive(Clone)]
```

Comment:
I'm guessing we need Clone here and a few other places because `Loader` trait requires it?

Reply:
Oh we need Clone here because it's required in the builder code.